haiku-apps/bescreencapture/bescreencapture-2.3.2.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
```

Comment:
I'm OK with the current `HOMEPAGE`, but I thought you might wish to change it to its `..`:
```diff
-HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
```